### ISSUE TITLE
Small changes to action text

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress-Future#419]
Change status to Draft
Change status to Private
Change status to Trash
Set taxonomy term(s) ... I'm not sure what this does? Is it different from "Add taxonomy term(s)"?

![Screenshot 2023-03-30 at 3 34 18 PM](https://user-images.githubusercontent.com/3868028/228944792-88f19dcf-6d87-4cc6-ae50-7bd0842dc484.png)

**Comment by andergmartins** [comment]
@stevejburge Yes, "Set term" replaces the current set of terms of a post. "Add term" appends another one.

Consider a post with the terms: `categ1`, `categ2`.
This would be the result for those actions:

* Set terms `categ3`, results on `categ3`.
* Add terms `categ3`, results on `categ1`, `categ2`, `categ3`.

**Comment by andergmartins** [comment]
@stevejburge thanks, yeah, those are more clear.

What about remove "post" from the others as well? Maybe change Delete post and others to:

* Delete
* Stick
* Unstick

**Comment by stevejburge** [comment]
@andergmartins Yes, good idea because the plugin should work with more than just posts
